### PR TITLE
Networked items fix

### PR DESCRIPTION
**Pull request by nayrpg**
This is our first pass of networked items. Ignore debugging messages they will get cleaned up once #95 is completed in a separate pr. This PR contains the following features: 

- Networked Use Item Inventory
- First Use Item: Bolt Cutters (this item is not networked and will be a separate instance for each player)
- The action slots no longer rely on the UI being present. 

Monkee!!!! 🦍

### CODE REVIEW

File: Assets/Prefabs/TestMap.prefab [hunk 1]
```diff
@@ -29623,3 +29625,149 @@ MonoBehaviour:
   networkParentLocalPosition: {x: 0, y: 0, z: 0}
   networkParentLocalRotation: {x: 0, y: 0, z: 0, w: 0}
   debug_mode: 0
+--- !u!1001 &427668312933732792
+PrefabInstance:
```
Comment (Feesh7) [inline Assets/Prefabs/TestMap.prefab]: What changed on the map?

File: Assets/Scenes/PracticeScene.unity [hunk 2]
```diff
@@ -847,85 +847,3 @@ Transform:
   m_Father: {fileID: 0}
   m_RootOrder: 0
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
---- !u!1 &2002111687
-GameObject:
-  m_ObjectHideFlags: 0
-  m_CorrespondingSourceObject: {fileID: 0}
```
Comment (Feesh7) [inline Assets/Scenes/PracticeScene.unity]: Also what changed in the menu?

Comment (nayrpg) [reply Assets/Scenes/PracticeScene.unity]: deleted the second camera

File: Assets/Prefabs/TestMap.prefab (hunk 1 above)
Comment (nayrpg) [reply Assets/Prefabs/TestMap.prefab]: Added the items on the map

**Pull request merged**